advocacy_docs/partner_docs/VeritasNetBackupforPostgreSQL/04-ConfiguringVeritasNetBackupforPostgreSQL.mdx

```diff
@@ -91,21 +76,21 @@ The steps below show an example of how to configure Veritas NetBackup for Postgr
     ```
 
 !!! Note
-    Value of **PGSQL_LIB_INSTALL_PATH** will be dependent on the version of EDB Postgres Advanced Server installed.
-    Values of **MASTER_SERVER_NAME** and **POLICY_NAME** parameters must match the names of your NetBackup Master Server and DataStore Policy respectively.
+    The value of `PGSQL_LIB_INSTALL_PATH` depends on the version of EDB Postgres Advanced Server installed.
+    Values of the `MASTER_SERVER_NAME` and `POLICY_NAME` parameters must match the names of your NetBackup master server and datastore policy, respectively.
 
-In the sample configuration file above, the values for **DB_USER**, **DB_PORT**, **DB_INSTANCE_NAME**, and **PGSQL_LIB_INSTALL_PATH** have been substituted for EDB Postgres Advanced Server as the default values are for Postgres.
-Refer to the [Veritas NetBackup for PostgreSQL Administrator's Guide](https://www.veritas.com/content/support/en_US/doc/129277259-150015228-0/v129276458-150015228) for detailed description of the parameters.
+In the sample configuration file, the values for `DB_USER`, `DB_PORT`, `DB_INSTANCE_NAME`, and `PGSQL_LIB_INSTALL_PATH` were for EDB Postgres Advanced Server as the default values are for Postgres.
```

Comment:
I stumbled over this rewrite. Maybe "the values for .... for EDB Postgres Advanced Server are set as the default values..." Or maybe "are used as the default values..."

Reply:
Yes, this was confusing. I revised this to what the correct meaning based on rereading the text above. I don't think we need the SME to confirm. This confusion stemmed from the use of the word "as" to mean "because," but it wasn't very clear anyhow.